ansible/library/cf_security_groups.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
+
+# Add security groups to CloudFormation template files.
+# This will modify the CloudFormation template files,
+
+import sys, json, glob
+from ansible.module_utils.basic import *
+from cfyaml import yaml
+
+
+def read_template(file_name):
+    with open(file_name, 'r') as f:
+        template = yaml.load(f)
+    return template
+
+
+def write_template(file_name, template):
+    with open(file_name, 'w') as f:
+        text = yaml.dump(template, default_flow_style=False)
+        f.write(text)
+
+
+def add_security_groups_to_elb(elb_resource, security_groups, template, debug_output):
+
+    for resource in template['Resources']:
+        if elb_resource in resource:
+            elb_security_groups = template['Resources'][elb_resource]['Properties']['SecurityGroups']
+            for secgrp in security_groups:
+                #debug_output.append('Adding: ' + secgrp + ' to ' + elb_resource)
+                elb_security_groups.append(secgrp)
+
+    return template
+
+
+def main():
+
+    module = AnsibleModule(
+      argument_spec = dict(
+        template_dir = dict(required=True, type='str'),
+        elb_resource = dict(required=True, type='str'),
+        security_groups = dict(required=True, type='list')
+      )
+    )
+
+    # for debug text
+    debug_output = []
```

Comment:
This is no longer needed once all commented out debug code is removed.

Reply:
Done